src/coreclr/jit/emitxarch.cpp

```diff
@@ -9493,7 +9507,7 @@ void emitter::emitDispIns(
             assert(IsThreeOperandAVXInstruction(ins));
             regNumber reg2 = id->idReg2();
             regNumber reg3 = id->idReg3();
-            if (ins == INS_bextr || ins == INS_bzhi)
+            if (ins == INS_bextr || ins == INS_bzhi || ins == INS_shrx || ins == INS_shlx || ins == INS_sarx)
             {
                 // BMI bextr and bzhi encodes the reg2 in VEX.vvvv and reg3 in modRM,
```

Comment:
Does the comment need updating or changing? it specifically mentions the set of things it used to apply to and no longer does.

Reply:
Added 3 shift instructions in the comment.